src/MonoTorrent/MonoTorrent.Client/PeerConnections/SocketConnection.cs

```diff
@@ -43,19 +43,19 @@ public class SocketConnection : IConnection
 
         public bool CanReconnect => !IsIncoming;
 
-        public bool Connected => Socket.Connected;
+        public bool Connected => SK.Connected;
```

Comment:
Can we leave it as `Socket` instead of renaming to `SK`?

Reply:
purely for I can set SK = null;